utils/channels-topics.js

```diff
@@ -0,0 +1,158 @@
+import { URL } from "url";
```

Comment:
lets follow best practice from https://github.com/asyncapi/template-for-generator-templates and have `helpers` and not `utils`